### PR TITLE
[NEXUS-7360,NEXUS-7737] OSGI bundle console UI, bundles in system-info report

### PR DESCRIPTION
**Pull request by jdillon**
Replaces plugin console with bundle console, information display only no operations yet to change state of bundles, etc.

https://issues.sonatype.org/browse/NEXUS-7360

Bundle list:

https://www.evernote.com/shard/s18/sh/2f95723b-28df-4e9c-8df0-17ee0346f250/b1c21e3ae71333474074eff02d52b523

Bundle detail:

https://www.evernote.com/shard/s18/sh/8feb27ce-a48f-4cc6-a9fd-905bc382c428/dfb3e7e54ede4335785b971339deff01

(a bit fugly atm, but was simplest to get something going, should refactor presentation of details)

Also replaces plugins with bundles in system information report:

https://issues.sonatype.org/browse/NEXUS-7737

https://www.evernote.com/shard/s18/sh/ac944b7e-7842-48ee-b457-4a5a3c7de451/17a6074ea1a78deeebc1b4522d3de02e

### CODE REVIEW

**Comment by jdillon** [comment]
http://bamboo.s/browse/NX3-OSSF150-2 :white_check_mark:

**Comment by jdillon** [comment]
http://bamboo.s/browse/NX3-OSSF153-2 :white_check_mark:

**Comment by adreghiciu** [comment]
+1

File: plugins/internal/nexus-atlas-plugin/src/main/java/org/sonatype/nexus/atlas/internal/SystemInformationGeneratorImpl.groovy [hunk 1]
```diff
@@ -47,22 +48,27 @@ class SystemInformationGeneratorImpl
...
-                                 final List<PluginIdentity> pluginIdentities)
+                                 final BundleContext bundleContext)
   {
     this.componentLookupHelper = checkNotNull(componentLookupHelper)
     this.applicationDirectories = checkNotNull(applicationDirectories)
     this.systemStatusProvider = checkNotNull(systemStatusProvider)
-    this.parameters = checkNotNull(parameters);
-    this.pluginIdentities = checkNotNull(pluginIdentities)
+    this.parameters = checkNotNull(parameters)
+    this.bundleContext = checkNotNull(bundleContext)
```
Comment (kellyrob99) [inline plugins/internal/nexus-atlas-plugin/src/main/java/org/sonatype/nexus/atlas/internal/SystemInformationGeneratorImpl.groovy]: doesn't look like we need to keep a reference to this beyond the lookup?

**Comment by kellyrob99** [comment]
+1

**Comment by cstamas** [comment]
+1

File: plugins/internal/nexus-atlas-plugin/src/main/java/org/sonatype/nexus/atlas/internal/SystemInformationGeneratorImpl.groovy (hunk 1 above)
Comment (mcculls) [inline plugins/internal/nexus-atlas-plugin/src/main/java/org/sonatype/nexus/atlas/internal/SystemInformationGeneratorImpl.groovy]: it's also used later on in the `bundleContext.bundles.each` loop, though that could always be replaced by `bundleService.selectBundles(null, true).each`

**Comment by mcculls** [comment]
+1

**Comment by jdillon** [comment]
manually merged

**Pull request closed without merging**